pr_check.sh

```diff
@@ -17,19 +17,24 @@
 # Options that must be configured by app owner
 # --------------------------------------------
 export APP_NAME="ccx-data-pipeline"  # name of app-sre "application" folder this component lives in
+# TODO set it to "insights-stage" once we have dvo-writer/dvo-extractor stage deploy config
+# TODO set it back to "insights-production" once we have dvo deployed to prod
+export REF_ENV="insights-stage"
 export COMPONENT_NAME="dvo-extractor"
 export IMAGE="quay.io/cloudservices/dvo-extractor"
-export COMPONENTS="dvo-extractor"  # space-separated list of components to laod
+export COMPONENTS="ccx-data-pipeline ccx-insights-results ccx-redis dvo-writer dvo-extractor insights-content-service insights-results-smart-proxy ccx-mock-ams"  # space-separated list of components to laod
```

Comment:
why do we need so many components?

Reply:
to run the pipeline tests. Or are we fine only with running dvo specific tests?

Reply:
@juandspy to run any usable iqe tests we need smart-proxy and other dependend services to provide the public API endpoints for testing.
We might just limit the test suite to the `dvo` marker then.

Reply:
do we need cache writer and redis too?

Reply:
or ccx-data-pipeline?

Reply:
probably not. ok, let's limit the scope and then see what services actually are needed for the dvo tests.